tasks/process_agent.py

```diff
@@ -12,9 +13,8 @@
 BIN_DIR = os.path.join(".", "bin", "process-agent")
 BIN_PATH = os.path.join(BIN_DIR, bin_name("process-agent", android=False))
 
-
 @task
-def build(ctx, race=False, incremental_build=False, puppy=False):
+def build(ctx, race=False, go110=False, incremental_build=False, puppy=False):
```

Comment:
Not a huge fan of switch flags that are this specific, go `1.10` will be a relic before we know it.

Reply:
I think we will want to remove this as soon as we can use `1.12` but we can rename the option